packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx

```diff
@@ -64,8 +64,11 @@ export default function usePlanBillingDescription( {
 		yearlyVariantPricing &&
 		( ! introOffer || introOffer.isOfferComplete )
 	) {
-		const yearlyVariantMaybeDiscountedPrice =
-			yearlyVariantPricing.discountedPrice?.monthly || yearlyVariantPricing.originalPrice?.monthly;
+		const yearlyVariantMaybeDiscountedPrice = Number.isFinite(
```

Comment:
NIT. Is there a reason we do `typeof` check below and `Number.isFinite` here?

Reply:
I would have preferred to use `Number.isFinite` everywhere but had to use the `typeof` check below to satisfy Typescript. I think the only difference is that `typeof` returns true for Infinity and NaN.